jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBResultMetadata.java

```diff
@@ -59,10 +59,7 @@ public String getColumnClassName(int arg0) throws SQLException {
   }
 
   @Override
-  public int getColumnCount() throws SQLException {
-    if (columnInfoList == null || columnInfoList.isEmpty()) {
-      throw new SQLException("No column exists");
-    }
+  public int getColumnCount() {
     return columnInfoList.size();
```

Comment:
What if columnInfoList is null? I think you can return 0 if it is null.

Reply:
I think it is okay to have null `columnInfoList`, which means some error occurs. If the size `columnInfoList` is 0, it means it is an empty list. What do you think?

Reply:
Actually, in the normal process, this couldn't be null, but in case others set a null in the future, check null is better